apps/maintenance/package.json

```diff
@@ -1,12 +1,34 @@
 {
   "name": "maintenance",
-  "version": "0.1.0",
+  "version": "0.0.0",
   "private": true,
   "scripts": {
-    "dev": "next dev --port 3003",
+    "dev": "next dev",
```

Comment:
Why are you removing the port? This way, `maintenaince` will run on the default one (3000), the one used by `web`.